standard/docs/en/implementation/amendments.md

```diff
@@ -16,21 +16,43 @@ The nature of a change can be made explicit using:
 
 * **The amendments** building block. This can contain an array of amendment explanations, and clearly identify the releases that contain before and after values. 
 
-## Worked example
+## Worked examples
 
-In the example below:
+Each example includes sample JSON files located at the bottom of the page. Use the combobox at the end of the page to display the files for each example and step.
```

Comment:
I think it would be better to thread the sample JSON files in between each step, as with the [easy releases example](https://standard.open-contracting.org/docs-easy-releases/en/implementation/easy-releases/) and use expanding/collapsing to highlight the relevant sections.